carbonmark-data/app/[locale]/trends/by-beneficiary/page.tsx

```diff
@@ -0,0 +1,5 @@
+import RetirementTrendsPage from "components/pages/RetirementTrendsPage";
+
+export default function RetirementTrendsByBeneficiaryPage() {
+  return <RetirementTrendsPage>By Beneficiary...</RetirementTrendsPage>;
```

Comment:
```suggestion
  return <RetirementTrendsPage>By Beneficiary…</RetirementTrendsPage>;
```

Please use `…` instead of `...` for better a11y. See [here](https://tosbourn.com/the-importance-of-using-the-ellipsis-character/) for more details